client/editEvents.js

```diff
@@ -201,10 +201,10 @@ Template.editEvents.helpers({
     return startDate.fromNow() + " (" + rangeStr + ")";
   },
   roundOpen: function() {
-    return this.status == wca.roundStatuses.open;
+    return this.isOpen();
```

Comment:
Ah, I like the parallelism as well. Is there a pending commit that removes `roundOpen`?